### DISCUSSION TITLE
Potential issue while read Sentinel3 OL_1_EFR geo coordinate

### DISCUSSION THREAD

**Discussion by yawen8617** [sertit/eoreader#136] [General]
Hi,

I'm recently using Eoreader to process Sentinel-3 OL_1_EFR data. When load data directly from a unzipped folder, the package mistakenly read `tie_geo_coordinates.nc` as `geo_coordinates.nc` when I try to load a band.

The example data is from [here](https://ladsweb.modaps.eosdis.nasa.gov/archive/allData/450/S3A_OL_1_EFR/2023/213/S3A_OL_1_EFR____20230801T164204_20230801T164504_20230802T171910_0180_101_354_2160_PS1_O_NT_003.zip).

The error message is:
```
File ~/gitlab/data-lake-env/lib/python3.9/site-packages/dask/array/core.py:3130, in normalize_chunks(chunks, shape, limit, dtype, previous_chunks)
   3122         raise ValueError(
   3123             "Input array has %d dimensions but the supplied "
   3124             "chunks has only %d dimensions" % (len(shape), len(chunks))
   3125         )
   3126     if not all(
   3127         c == s or (math.isnan(c) or math.isnan(s))
   3128         for c, s in zip(map(sum, chunks), shape)
   3129     ):
-> 3130         raise ValueError(
   3131             "Chunks do not add up to shape. "
   3132             "Got chunks=%s, shape=%s" % (chunks, shape)
   3133         )
   3134 if allints or isinstance(sum(sum(_) for _ in chunks), int):
   3135     # Fastpath for when we already know chunks contains only integers
   3136     return tuple(tuple(ch) for ch in chunks)

ValueError: Chunks do not add up to shape. Got chunks=((19897850,),), shape=(314930,)
```

Looks like the issue is from [this function](https://github.com/sertit/eoreader/blob/062e104513b3bb723bc22782fcd293d49d1915cb/eoreader/products/optical/s3_olci_product.py#L449). It handles zipped and unzipped path. If path is a zipped file, it use `path.get_archived_path` function from package `sertit`. It will return only one file path end with `geo_coordinates.nc`. If it's a unzip folder, it uses `Path.glob` to find all files endwith `geo_coordinates.nc`. It can be `geo_coordinates.nc` or `tie_geo_coordinates.nc`. 

Looks like this is a potential bug. Could you look into that? 

Thank you!

**Comment by remi-braun** [comment]
Thanks for this, it seems like an issue yes :)
Good investigation, I'll update it asap!

**Reply by remi-braun** [reply]
See issue #137
